cpp/dolfinx/fem/Form.h

```diff
@@ -598,7 +405,8 @@ class Form
         {
           // There will only be one pair for an exterior facet integral
           std::array<std::int32_t, 2> pair
-              = get_cell_local_facet_pairs<1>(f, f_to_c->links(f), *c_to_f)[0];
+              = mesh::get_cell_local_facet_pairs<1>(f, f_to_c->links(f),
```

Comment:
Not changed in this PR, but docstring for `get_cell_local_facet_pairs` is wrong.